packages/dd-trace/test/opentracing/propagation/text_map.spec.js

```diff
@@ -117,6 +117,21 @@ describe('TextMapPropagator', () => {
       expect(carrier).to.have.property('x-b3-flags', '1')
     })
 
+    it('should inject the traceparent header', () => {
+      const carrier = {}
+      const spanContext = new SpanContext({
+        traceId: id('0000000000000123'),
+        spanId: id('0000000000000456'),
+        sampling: {
+          priority: USER_KEEP
+        }
+      })
+
+      propagator.inject(spanContext, carrier)
+
+      expect(carrier).to.have.property('traceparent', '01-0000000000000123-0000000000000456-01')
```

Comment:
Is this compliant? Shouldn't the trace ID be padded to 32 characters?

Reply:
Correct, it's a miss on my side.